src/dev/pages/stack/stack.ts

```diff
@@ -0,0 +1,54 @@
+// Styles
+import './stack.scss';
+import '@tylertech/forge/stack/stack.scss';
+
+// Components
+import '@tylertech/forge/stack';
+
+import '$src/shared';
+import type { ISwitchComponent } from '@tylertech/forge/switch';
+import { ISelectComponent, IStackComponent, ITextFieldComponent } from '@tylertech/forge';
+
+
+const inlineToggle = document.querySelector('#inline-switch') as ISwitchComponent;
+const wrapToggle = document.querySelector('#wrap-switch') as ISwitchComponent;
+wrapToggle.disabled = true;
```

Comment:
FYI, you can set `disabled: true` on the option configuration to default it to disabled.